packages/extension/src/view/devtools/components/settings/components/informationContainer.tsx

```diff
@@ -64,15 +65,19 @@ const InformationContainer = () => {
   const handleCopy = useCallback(() => {
     setCopying(true);
 
-    let clipboardText = `**Open Tabs:** ${currentTabs}\n`;
+    let clipboardText = `**${I18n.getMessage('openTabs')}:** ${currentTabs}\n`;
```

Comment:
Do we need translated details here? AFAIK this is for developers to replicate the environment to reproduce the breakage.